README.md

```diff
@@ -8,8 +8,9 @@
 # ARCS
 
 Scaffolding genome sequence assemblies using 10X Genomics GemCode/Chromium data. 
-ARCS can be run in 2 modes:
-* [ARCS](https://doi.org/10.1101/100750) (default) uses alignments of linked reads to the input contigs 
+ARCS can be run in 3 modes:
+* [ARCS](https://doi.org/10.1101/100750) (default) uses alignments of linked reads to the input contigs
+* ARCS-long (`arcs-long`) uses alignments of long reads to the input contigs (only available when running the ARCS+LINKS pipeline) 
```

Comment:
```suggestion
* ARCS-long (`arcs-long`) uses alignments of long reads to the input contigs (only available when running ARCS in default mode) 
```
Just because people might be confused, since `ARKS` and `ARCS` are all in this one repo now